### ISSUE TITLE
pass password in a url

### ISSUE THREAD

**Issue by MichaelRumpler** [connectbot/connectbot#398]
I try to open ConnectBot with a link on a web page. It opens and connects to the server when my link looks like

               ssh://user@host:port/

But I still have to type the password. How can I pass it to ConnectBot?
This does not work:

               ssh://user:pwd@host:port/

Is there also a way to pass a key file (either as base64 or the file path/url)?

We have a global web page which only the administrators can access. From this web page, they should be able to connect to all servers. As they already authenticated when they opened the web page, there’s no need to ask for the password again. This should be a single sign on to all servers. We can also change passwords easily without having to tell everybody.

**Comment by MichaelRumpler** [comment]
This is currently not supported. Changes would have to be made (at least) in
- HostBean
- transport.SSH.authenticate

I don't know if you consider this a security risk. IMHO it is not, but I don't want to implement something which will not be regarded anyway, so I won't do it yet.

**Comment by keks** [comment]
I don't think this is a good idea. You should not never have your password as plain text anywhere. If you want automatic login, use public keys. On a side note, the openssh `ssh` client doesn't allow `echo passwd | ssh user@host`.
